web/src/components/dashboard/SidebarCardsContainer.test.tsx

```diff
@@ -77,51 +77,6 @@ describe("<SidebarCardsContainer />", () => {
   };
 
   describe("nudges", () => {
-    it("renders the welcome card", () => {
-      const sidebarCards = {
-        welcome: generateSidebarCardContent({ contentMd: "WelcomeCardContent" }),
-      };
-
-      useMockBusiness({ preferences: generatePreferences({ visibleSidebarCards: ["welcome"] }) });
-
-      renderPage({ sidebarCards });
-      expect(screen.getByText("WelcomeCardContent")).toBeInTheDocument();
-    });
-
-    it("renders a card in the top section if set to above-opportunities", () => {
```

Comment:
these two tests regarding above opportunities and below opportunities should be kept because they are not specific to the welcome card but rather testing logic in regards customizing the card position

Reply:
So a different card should be put in the array since the welcome card no longer exists. Correct?